pages/1.11/installing/production/system-requirements/index.md

```diff
@@ -62,7 +62,7 @@ There are many mixed workloads on the masters. Examples of mixed workloads on th
   | _/var/lib/dcos/cockroach_ | CockroachDB [enterprise type="inline" size="small" /] |
   | _/var/lib/dcos/navstar_ | for Mnesia database |
   | _/var/lib/dcos/secrets_ | secrets vault [enterprise type="inline" size="small" /] |
-  | _/var/lib/dcos/exec_ | Temporarily needed files from various DC/OS services. _/var/lib/dcos/exec_ must not be on a volume which is mounted `noexec`. |
+  | _/var/lib/dcos/exec_ | Temporary files required by various DC/OS services. The _/var/lib/dcos/exec_ directory must not be on a volume which is mounted with the `noexec` option. |
```

Comment:
@lsgunnlsgunn "Temporarily needed files" was a considered choice. The files are not temporary - nothing is deleting them. Can that change be reverted?

Reply:
Okay, how about "Files that are temporarily needed by various DC/OS services."
Hyphenated modifier words like "Temporarily-needed " are generally a bit hard to parse. Let me know and I'll fix it.